opentelemetry-instrumentation/src/opentelemetry/instrumentation/utils.py

```diff
@@ -60,3 +61,12 @@ def unwrap(obj, attr: str):
     func = getattr(obj, attr, None)
     if func and isinstance(func, ObjectProxy) and hasattr(func, "__wrapped__"):
         setattr(obj, attr, func.__wrapped__)
+
+
+def remove_url_credentials(url: str) -> URL:
```

Comment:
Should this return a string instead of a `URL`?  It seems like it works fine for now, presumably because of implicit stringification of the `URL` where it's used, but I'm not sure what impact changing the type of this value may have.

Reply:
A string would make more sense however the existing implementation of one of the instrumentations (aiohttp) takes advantage of the `URL` format and that's why it made sense to not be destructive of that implementation.  I'll explore if returning a string instead would be possible before submitting upstream.

Reply:
It seems like you've changed this to return a str now, right? Just to confirm